stories/LogoSubtitleBody.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { storiesOf, addDecorator } from '@storybook/react'
+import Theme from './theme'
+import { Grid, Col, Row } from '../src/components/grid'
+import WhyOpenSource from '../src/components/OpenSource/WhyOpenSource'
+
+import icon from './assets/community_support.svg'
+
+addDecorator(Theme)
+
+const title = 'Community support'
+const body =
+  'Open source technologies are supported by knowledgeable and resourceful communities. Their collective contributions result in more robust, innovative and faster solutions.'
+
+storiesOf('Logo, Subtitle & Body', module).add('on white Background', () => {
```

Comment:
Could we have the name of this story reflect the name of the React component? :pray: 
i.e. `LogoSubtitleBody` rather than comma separated

That way, someone (e.g. a designer) just looking at storybook already knows exactly what the component is called in our codebase